scripts/MCUXpresso_Config_Tools/darwin-amd64/launch-MCUXpressoConfigTools

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+# Script launch MCUXpresso Config Tools in OpenCMSIS generator mode.
+
+echo Platform is not supported for MCUXpresso Config Tools in OpenCMSIS generator mode!
+exit 1
```

Comment:
Please add a newline in the files marked by GH as having "No newline at end of file".

Reply:
Added a newline into the launch files